MapboxNavigation/RouteVoiceController.swift

```diff
@@ -215,16 +214,19 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
         // If there is no `upComingStep`, there definitely should not be a followOnStep.
         // This should be caught above.
         let upComingInstruction = routeStepFormatter.string(for: routeProgress.currentLegProgress.upComingStep, legIndex: upcomingLegIndex, numberOfLegs: numberOfLegs, markUpWithSSML: markUpWithSSML)!
-        let stepDistance = routeProgress.currentLegProgress.upComingStep!.distance
+        let upcomingStepDuration = routeProgress.currentLegProgress.upComingStep!.expectedTravelTime
         let currentInstruction = routeStepFormatter.string(for: routeProgress.currentLegProgress.currentStep, legIndex: routeProgress.legIndex, numberOfLegs: numberOfLegs, markUpWithSSML: markUpWithSSML)
         let step = routeProgress.currentLegProgress.currentStep
         var text: String
         
+        // Prevent back to back instructions by adding a little more wiggle room
+        let linkedInstructionMultiplier = RouteControllerHighAlertInterval * 1.2
```

Comment:
This variable is of type `TimeInterval`, so it’s an “interval”, not a “multiplier” (which implies no units).

Also factor out 1.2 (which is the real multiplier) into a constant. Provide a documentation comment that explains why 1.2 is the right value for this constant.